src/main/resources/specs/swagger.yml

```diff
@@ -0,0 +1,81 @@
+openapi: 3.0.1
```

Comment:
Why did you decide to store API spec YAML in repository?

Reply:
It would be nice to store the API documentation in a central place like SwaggerHub - we could then have an integration test that reads the contract/specification and validates our implementation against it. I decided to keep this in here to be in line with the other APIs at CH. My plan is to at least create a test that validates our implementation matches this. This could be done later though

Reply:
Have you added that test that uses swagger schema?

Reply:
I do not see any test using it so please remove. At this point until used it adds no benefit but brings maintenance cost and confusion when there go out of sync.

Reply:
ok, will remove this for now and add it again in the future with a test